src/Controller/MenuController.java

```diff
@@ -444,15 +444,12 @@ public void loadCalendar() {
 		// =================
 		// Populate Agenda:
 		ArrayList<Event> calendar = MainController.getSPC().getPlanner().getCurrentStudyProfile().getCalendar();
-		
-        //Counter to create unique ICS file names for export
-        int counter = 0;
+		//Counter to create unique ICS file names for export
+		int counter = 0;
         //Creation of ICS export factory
         ICalExport iExport = new ICalExport();
         //Preparation of export directory
         iExport.iCalSetup();
-		
```

Comment:
One of these should likely have remained as a blank line to maintain readability.